index.html

```diff
@@ -37,8 +37,128 @@ <h1>I'm Sohail Glad to see you!</h1>
+                <img src="./images/Snapshoot Portfolio.png" alt="">
+                <div class="card-work work">
+                    <h3>Multi-Post <br> Stories</h3>
+                    <ul class="proj">
+                        <li><a href="#">CANOPY</a></li>
+                        <li><div class="dot"></div></li>
+                        <li>Back End Dev</li>
+                        <li><div class="dot"></div></li>
+                        <li>2015</li>
+                    </ul>
+                    <p class="pro">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="language">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                    </ul>
+                    <a href="#" class="button">See Project</a>
+                </div>
+            </div>
+            <div class="works">
+                <img src="./images/Snapshoot Portfolio-1.png" alt="">
+                <div class="card-work work">
+                    <h3>Tonic</h3>
+                    <ul class="proj">
+                        <li><a href="#">CANOPY</a></li>
+                        <li><div class="dot"></div></li>
+                        <li>Back End Dev</li>
+                        <li><div class="dot"></div></li>
+                        <li>2015</li>
+                    </ul>
+                    <p class="pro">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="language">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                    </ul>
+                    <a href="#" class="button">See Project</a>
+                </div>
+            </div>
+            <div class="works">
+                <img src="./images/Snapshoot Portfolio-2.png" alt="">
+                <div class="card-work work">
+                    <h3>Multi-Post <br> Stories</h3>
+                    <ul class="proj">
+                        <li><a href="#">CANOPY</a></li>
+                        <li><div class="dot"></div></li>
+                        <li>Back End Dev</li>
+                        <li><div class="dot"></div></li>
+                        <li>2015</li>
+                    </ul>
+                    <p class="pro">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="language">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                    </ul>
+                    <a href="#" class="button">See Project</a>
+                </div>
+            </div>
+        </section>
+<!-- ------------------------------------------------about me--------------------------------         -->
+        <section class="about">
```

Comment:
- [ Required ] As requested by a previous reviewer please make sure that the `About Myself` section has a `border-radius` on the top right corner of your design. Please see the screenshot below for more information on this.

## Figma template
<img src="https://user-images.githubusercontent.com/16392046/205234730-e5b7c4a9-9a97-4e54-8a51-fa8d5fafbf2b.png" width="350px" />

## Your Implementation
<img src="https://user-images.githubusercontent.com/16392046/205235025-f835ff18-107e-4795-b282-0f88c8320fe0.png" width="350px" />